### ISSUE TITLE
Does pytest-bdd-html support pytest-xdist?

### ISSUE THREAD

**Issue by jun-x-nova** [slavos1/pytest-bdd-html#18]
Running pytest-bdd (cellenium script) in parallel with pytest-xdist does not collect the results in the report.

**Comment by slavos1** [comment]
Hello @jun-x-nova, thanks for using the plugin and finding time to raise an issue.

There's already #6 which is still open.

P.S. Rereading your question, did you mean pytest-bdd-html (this plugin) or [pytest-bdd](https://github.com/pytest-dev/pytest-bdd)?

**Comment by slavos1** [comment]
Closing as a duplicate of #6.

**Comment by jun-x-nova** [comment]
I have confirmed the contents. Do you have any future plans?

**Comment by slavos1** [comment]
Yes, I want to at least update documentation if not possible to run with pytest-xdist.
